client/js/components/user/orgaRegisterForm/OrgaRegisterForm.vue

```diff
@@ -49,37 +49,52 @@
           </fieldset>
 
           <fieldset>
-            <legend :class="prefixClass('font-size-medium is-label u-mb-0_25')">
+            <legend :class="prefixClass('font-size-medium is-label u-mb-0_5')">
               {{ Translator.trans('organisation.type') }}
             </legend>
-            <div :class="hasPermission('feature_identity_broker_login') ? prefixClass('space-stack-xs') : prefixClass('o-form__group')">
-              <dp-checkbox
-                id="orgatype_invitable_institution"
-                data-cy="orgatype_institution"
-                :class="prefixClass('o-form__group-item')"
-                :label="{
-                  text: Translator.trans('invitable_institution')
-                }"
-                name="r_orgatype[]"
-                value-to-send="OPSORG" />
-              <dp-checkbox
-                id="orgatype_municipality"
-                data-cy="orgatype_municipality"
-                :class="prefixClass('o-form__group-item')"
-                :label="{
-                  text: Translator.trans('municipality')
-                }"
-                name="r_orgatype[]"
-                value-to-send="OLAUTH" />
-              <dp-checkbox
-                id="orgatype_planningagency"
-                data-cy="orgatype_planningagency"
-                :class="prefixClass('o-form__group-item')"
-                :label="{
-                  text: Translator.trans('planningagency')
-                }"
-                name="r_orgatype[]"
-                value-to-send="OPAUTH" />
+            <div :class="hasPermission('feature_identity_broker_login') ? prefixClass('space-stack-xs') : prefixClass('o-form__group flex-col')">
+              <div :class="prefixClass('u-mb-0_75')">
+                <dp-checkbox
+                  id="orgatype_invitable_institution"
+                  data-cy="orgatype_institution"
+                  :class="prefixClass('o-form__group-item')"
+                  :label="{
+                    text: Translator.trans('invitable_institution')
```

Comment:
Does it also work when placing the hints directly within the labels? this way they'd be clickable as well.

```suggestion
                    text: Translator.trans('invitable_institution'),
                    hint: Translator.trans('register.institution.hint')
```

Reply:
thx for the hint. It works fine. done here https://github.com/demos-europe/demosplan-core/pull/2045/commits/1b12e7682333daa0429f0039368fe118123c6e39